### ISSUE TITLE
Error with cellInfo

### ISSUE THREAD

**Issue by KatjaRM** [aertslab/SCENIC#97]
Hi Team,

Thank you for developing this software and for the tutorial!

I have been running into an issue with code that uses cellInfo, eg. 
`plotTsne_compareSettings(fileNames, scenicOptions1, showLegend=FALSE, varName="CellType", cex=.5)` returns `Cannot find cell information named 'CellType (it should be a column of the 'cellInfo').`

However, my scenicOptions file does contain cellinfo:
`head(scenicOptions@inputDatasetInfo$cellInfo)`

                 CellType nGene  nUMI
ACGCAGCGTTCCACGG      GSC  2644 11828
ACGCAGCTCTGGAGCC      GSC  3235 17779
GATCGCGCACGTCAGC      GSC  3257 15371
GGAGCAAGTCTGGTCG      GSC  2827 10936
AACCGCGGTCTAGTGT      pFC  1622  6345
ACACCAAAGAGTAAGG      pFC  2313 12250

I have imported the cellInfo from a Seurat file, since `get_cellAnnotation(loom)` returned an error (Error in args_regularity_evaluation(args = args, ds_dims = simple_extent$dims,  :Number of arguments not equal to number of dimensions: 1 vs. 2) which I was not able to figure out.

Do you have any idea how I could fix this issue? Would be grateful for any help!

Best,

Katja

**Comment by KatjaRM** [comment]
Edit: found the issue... Sorry!

**Comment by Thegreatjoyce** [comment]
> Edit: found the issue... Sorry!

? I met the same situation, how did you solve it?

**Comment by BioInfoUCI** [comment]
Hello, I have the same issue. @KatjaRM, could you please let us know how you fixed it? Thank you

**Comment by KatjaRM** [comment]
Hi!
Sorry for the late reply.
I think at that time I had simply forgotten a step but what actually works way better for me is just reading in the information from the Seurat file. So if you have clustering information from Seurat try this:

cellInfo<- data.frame(seuratCluster=Idents(SeuratObject))
nGene <- SeuratObjec@meta.data$nGene
nGene <- as.data.frame(nGene)
nUMI <- SeuratObjec@meta.data&nUMI
nUMI <- as.data.frame(nUMI)
cellInfo["nGene"] <- nGene
cellInfo["nUMI"] <- nUMI

Hope this works for you!

Katja

**Comment by lelesama** [comment]
> Hello, I have the same issue. @KatjaRM, could you please let us know how you fixed it? Thank you
Hello, I have the same issue.  could you please let us know how you fixed it? Thank you

**Comment by lelesama** [comment]
> > Edit: found the issue... Sorry!
> 
> ? I met the same situation, how did you solve it?
Hello, I have the same issue. @KatjaRM, could you please let us know how you fixed it? Thank you

**Comment by lelesama** [comment]
> > Edit: found the issue... Sorry!
> 
> ? I met the same situation, how did you solve it?

Hello, I have the same issue. 
could you please let us know how you fixed it? Thank you

**Comment by KatjaRM** [comment]
Please see comment above from Jan 31st :)

**Comment by lelesama** [comment]
> Please see comment above from Jan 31st :)
> If this does not fix the issue for you I recommend opening another issue with the team. Hope you figure it out!

I just  don't understand  the step  forgotten you said  ,its difference between your issue and your solution,Thank you

**Comment by KatjaRM** [comment]
I am not sure which step I forgot. If running the tutorial steps for you does not work (https://rdrr.io/github/aertslab/SCENIC/f/vignettes/SCENIC_Running.Rmd), please open a new issue, since this is issue is actually closed and I am not sure anyone who can actually help you is reading it.

**Comment by XinYu-pumch** [comment]
@KatjaRM  

I was running another project in which i tried to convert a Seruat object into h5ad file and came across the same problem ------"args_regularity_evaluation(args = args, ds_dims = simple_extent$dims, :Number of arguments not equal to number of dimensions: 1 vs. 2) "

Finally I found wasy to cope with it. 

It was because **there was a data.frame in my Seurat object@meta.data** and when I deleted  (by setting "object@meta.data$xxx <- NULL") the code run without bugs.
